src/robot/libraries/Process.py

```diff
@@ -439,11 +440,15 @@ def wait_for_process(self, handle=None, timeout=None, on_timeout='continue'):
         | ${result} =                 | Wait For Process | timeout=1min 30s | on_timeout=kill |
         | Process Should Be Stopped   |                  |                  |
         | Should Be Equal As Integers | ${result.rc}     | -9               |
+
+        New in Robot Framework ``v3.2``: Falsy, negative, and zero values are also ignored.
```

Comment:
Great that you added a note about version when functionality was enhanced. This is slightly different style than we've used elsewhere with such notes, though. As mentioned above, this info could also be explained elsewhere.

Reply:
I don't think I came up with it myself. I remember seeing it in some "Libdoc" generated documentation, hence I tried it. Maybe it wasn't official RF documentation though.

It is also more clean to see expected functionality vs changelog separately, instead of intertwined with each other.

I can amend if you insist.

Reply:
In Robot's docs we've used two approach to specify when a certain feature has been added.
- When explaining the feature, use something like `Starting from Robot Framework 3.2, it is possible to do X.`.
- At the end have a note like `Doing X is a new feature in Robot Framework 3.2.` or possibly a bit shorted `Doing X is new in Robot Framework 3.2.`.

We haven't used  the `v` prefix with the version or added formatting for it. Definitely not a big deal but I really like consistency.

Reply:
_(Done locally)_